packages/loot-core/src/platform/client/undo/index.d.ts

```diff
@@ -2,12 +2,15 @@ export type UndoState = {
   id?: string;
   url: unknown;
   openModal: unknown;
-  selectedItems: unknown;
+  selectedItems: {
+    name: string;
+    items: Set<string>;
+  } | null;
 };
 
-export function setUndoState(
-  name: keyof Omit<UndoState, 'id'>,
-  value: unknown,
+export function setUndoState<K extends keyof Omit<UndoState, 'id'>>(
+  name: K,
+  value: State[K],
```

Comment:
Passes typecheck, so maybe I'm crazy -- but I don't see where `State` is imported. 
Should it be the new type you [just added](https://github.com/actualbudget/actual/pull/1479/files#diff-724ece9c0e81f198462ea9728a2be6925e73ee6661540682cb148abcc0198698R30)?

Reply:
Good catch! Fixed it here: https://github.com/actualbudget/actual/pull/1479/commits/81c95f7decc8f4dca149578d2681eebb2cff8ea7